gen/libdir/forbidden_ifetch.conf

```diff
@@ -0,0 +1,14 @@
+-arch AArch64
+-variant self
+
+-relax RfeIP [DpCtrlIsbdR, I] [DC.CVAUn, FrePI, PodWW, DSB.ISH, IC.IVAUp] [FrePI PodW* DC.CVAUp DSB.ISH IC.IVAUp DSB.ISH ISB] [FrePI PosW* DC.CVAUp DSB.ISH IC.IVAUp DSB.ISH ISB] [ Pod*R DSB.ISH DC.CVAUn DSB.ISH IC.IVAUn DSB.ISH ISB FrePI] [ DpCtrlIsbdR DC.CVAUn DSB.ISH IC.IVAUn DSB.ISH ISB FrePI] [ Pod*R DSB.ISH IC.IVAUn DSB.ISH ISB FrePI PodW* DC.CVAUp DSB.ISH] [PodRR FrePI PodW* DC.CVAUp DSB.ISH IC.IVAUp DSB.ISH]
+
+-safe Rfe Fre DMB.ISH RfeII FreII DMB.STdWW DpAddrdR DpAddrdW DpAddrsR DpAddrsW DpDatadW DpDatasW DpCtrldW DpCtrlsW DpCtrlIsbdR [DC.CVAUn Pos**  DC.CVAUp] [DpAddrdR, DC.CVAUn] [DpAddrdR, IC.IVAUn] [DpAddrsR, DC.CVAUn] [DpAddrsR, IC.IVAUn] [Pos**, IC.IVAUn, DSB.ISH,  DC.CVAUn] [Pos**, IC.IVAUn, DSB.ISH, IC.IVAUn] [Pos**, DC.CVAUn, DSB.ISH, DC.CVAUn] [Pod**, DC.CVAUn, DSB.ISH, IC.IVAUn] [Pod**, IC.IVAUn, DSB.ISH] [Pod**, DC.CVAUn, DSB.ISH] [Pod**, IC.IVAUn, DSB.ISH,  DC.CVAUn] [Pod**, IC.IVAUn, DSB.ISH, IC.IVAUn] [Pod**, DC.CVAUn, DSB.ISH, DC.CVAUn] [Pos**, DC.CVAUn, DSB.ISH, IC.IVAUn] [Pos**, IC.IVAUn, DSB.ISH] [Pos**, DC.CVAUn, DSB.ISH]
```

Comment:
yes, I think you are right. If the configuration file was already generating only forbidden tests, then removing anything from the safe list will just reduce the number of tests generated. 

I thought having the DC/IC relaxations in the safe list would allow for more variety in the generated tests, but I agree it would be redundant for forbidding tests.

Did you want these removed because it would make the configuration file simpler? I don't mind removing them if you think they are not necessary

Reply:
At the time, my thinking was that the interesting forbidden tests would be generated without those other relaxations, and I was not sure about the value of tests that those other relaxations would add. It is good to generate more tests, but only if that does not prevent the useful tests from getting generated (by delaying them forever).

If extra relaxations do not cause concerns about performance of `diy7`, I suppose they are fine to keep.

Reply:
In that case I will keep them in. As far as I can tell, keeping them in isn't preventing useful tests from being generated

Reply:
OK, thanks for checking -- please consider this resolved. :-)